### DISCUSSION TITLE
Welcome to drove-orchestrator Discussions!

### DISCUSSION THREAD

**Discussion by santanusinha** [PhonePe/drove-orchestrator#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Vishdev63** [comment]
Hello @santanusinha 👋, I am excited to learn about Drove's architecture. Having used Kubernetes, I am curious about how Drove differs from it. I agree that Drove may integrate better with PhonePe's existing technology stack. Could you please provide insights from an architectural perspective?

**Reply by santanusinha** [reply]
From an architecture perspective, it's quite simple. One of the core things we were trying for is less moving parts and Drove has only 2, the controllers and the executors. Please go through [this page](https://phonepe.github.io/drove-orchestrator/cluster/cluster.html) to understand more about the architecture. You can approach a very well performing and stable cluster with minimal configuration and therefore has low startup cost and upfront investment. Some of the other features are mentioned in the introduction page as well. You can create a Drove cluster without needing almost any infrastructure changes into your existing DC or VPC in a short time and start running containers on it. NUMA based memory and CPU pinning come out of the box without any configuration needed. You should get quite a bit of performance boost if you're not running containers in a cpu and memory optimized manner currently.

I think a better way to think about drove is to compare it to Mesos/ECS etc rather than kubernetes. IMO, Kubernetes is a much more evovled product meant to build end to end infrastructure, container orchestration is a part of it.
